frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -37,6 +37,8 @@ const PREMIUM_ACTIVITIES = new Set([
   "enabled_macos_setup_end_user_auth",
   "disabled_macos_setup_end_user_auth",
   "tranferred_hosts",
+  "enabled_windows_mdm_migration",
+  "disabled_windows_mdm_migration",
```

Comment:
Not sure this is still technically required (PREMIUM_ACTIVITIES seems to be used only for the sandbox, which isn't a thing anymore IIRC?), but I think there's no harm in adding those.